### PR TITLE
Upgrade to PS 0.15.4 and switch to Spago.

### PR DESCRIPTION
**Pull request by aranchelk**

### CODE REVIEW

File: src/Pako.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+import pako from 'pako';
 
-exports.deflateImpl = function(options) {
-  return function(bytes) {
-    return function() {
-      return pako.deflate(bytes, options).buffer;
-    };
-  };
-};
+export const hello = () => console.log("hello");
```
Comment (jacereda) [inline src/Pako.js]: I guess this could removed?

**Review by jacereda — CHANGES_REQUESTED** [review]
Thanks for the PR. Would you be willing to maintain the package?

File: src/Pako.js (hunk 1 above)
Comment (aranchelk) [reply src/Pako.js]: That's embarrassing, I'm still getting used to ES6 module syntax, I must have pasted that in as a reference and forgotten about it. Thanks.

**Comment by aranchelk** [comment]
> Thanks for the PR. Would you be willing to maintain the package?

Sure. I don't mind at all.

**Pull request merged**